tests/tests.pro

```diff
@@ -5,12 +5,10 @@ PACKAGENAME = nemo-qml-plugin-systemsettings
 QT += testlib qml dbus systeminfo
 QT -= gui
 
-system(sed -e s/@PACKAGENAME@/$${PACKAGENAME}/g tests.xml.template > tests.xml)
-
 TEMPLATE = app
 TARGET = ut_diskusage
 
-target.path = $$[QT_INSTALL_LIBS]/$${PACKAGENAME}-tests
+target.path = /opt/tests/$${PACKAGENAME}-tests
 
 xml.path = /usr/share/$${PACKAGENAME}-tests
```

Comment:
This is still the old directory while the .spec file looks for it in the spec. Change this also to the same path as above.

Reply:
Fixed this as well :)